app/src/main/java/org/kiwix/kiwixmobile/main/KiwixWebViewClient.java

```diff
@@ -108,9 +114,30 @@ private void inflateHomeView(WebView view) {
     LayoutInflater inflater = LayoutInflater.from(view.getContext());
     home = inflater.inflate(R.layout.content_main, view, false);
     callback.setHomePage(home);
+
     if (sharedPreferenceUtil.nightMode()) {
+
       ImageView cardImage = home.findViewById(R.id.content_main_card_image);
-      cardImage.setImageResource(R.drawable.ic_home_kiwix_banner_night);
+      ImageView sideImage = home.findViewById(R.id.content_side_image);
+      AppCompatButton downloadButton = home.findViewById(R.id.content_main_card_download_button);
+
+      downloadButton.setTextColor(Color.parseColor("#000000"));
+      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+        downloadButton.setBackgroundTintList(ColorStateList.valueOf(
+            KiwixApplication.getInstance().getResources().getColor(R.color.complement_blue800)));
+      }
+
+      ConstraintLayout constraintLayout = home.findViewById(R.id.constraint_main);
+      constraintLayout.setBackgroundResource(R.drawable.back_cover_night);
+      sideImage.setImageResource(R.drawable.home_side_cover_night);
+
+      if (KiwixApplication.getInstance().getResources().getConfiguration().orientation ==
+          Configuration.ORIENTATION_PORTRAIT) {
+        cardImage.setImageResource(R.drawable.kiwix_vertical_logo_night);
+      } else if (KiwixApplication.getInstance().getResources().getConfiguration().orientation ==
+          Configuration.ORIENTATION_LANDSCAPE) {
```

Comment:
As I have told you before you should not check the orientation dynamically. Instead you should have the same name of the drawables for both the configuration and move the drawables for landscape to folders `drawable-land-hdpi`, `drawable-land-mdpi`, etc.

Reply:
@abdulwd Made all the changes.